### PR TITLE
Improve Readme

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,31 +37,29 @@ Two APIs are provided
 - Or they can use the context-based API similar to Matplotlib's pyplot, which
   provides sensible default choices for most parameters.
 
-Getting Started
----------------
+## Trying it online
 
-### Try it online with [Binder](http://mybinder.org/)
+To try out xtensor interactively in your web browser, just click on the binder
```
Comment (dmadeka) [inline README.md]: :)

File: README.md [hunk 2]
```diff
@@ -27,31 +37,29 @@ Two APIs are provided
...
+link:
 
-[![Binder](https://img.shields.io/badge/launch-binder-brightgreen.svg)](https://mybinder.org/v2/gh/bloomberg/bqplot/0.10.4-binder2?filepath=examples/Index.ipynb)
+[![Binder](docs/source/binder-logo.svg)](https://mybinder.org/v2/gh/QuantStack/xtensor/0.16.4?filepath=notebooks/xtensor.ipynb)
```
Comment (mwcraig) [inline README.md]: Looks like this (and the text above it) is pointing to the wrong binder...

File: README.md [hunk 3]
```diff
@@ -27,31 +37,29 @@ Two APIs are provided
...
 
-
 ### Installation
 
 Using pip:
 
 ```
 $ pip install bqplot
-$ jupyter nbextension enable --py --sys-prefix bqplot  # can be skipped for notebook version 5.3 and above
 ```
```
Comment (mwcraig) [inline README.md]: Maybe add jupyter-lab instructions here?

Comment (SylvainCorlay) [reply README.md]: Thanks, done!

**Pull request merged**